blockchains/ethereum/assets/0xFe6e2CBd2E28b3A6F43aa17f58d9AC5D6fA19B99/info.json

```
@@ -3,9 +3,9 @@
     "symbol": "KET",
     "type": "ERC20",
     "decimals": 18,
-    "description": "-",
-    "website": "",
+    "description": "Community Currency",
+    "website": "unknown",
```

```suggestion
    "website": "-",
```

Keep it as if there is no website